src/operation.rs

```diff
@@ -1,4 +1,4 @@
-//! Implementation of primitive operations.
+//! Implementat Closure::atomic_closureon of primitive operations.
```

Comment:
Let me assume that this is a wild copy/paste

```suggestion
//! Implementation of primitive operations.
```